src/.gitignore

```diff
@@ -0,0 +1,6 @@
+tests/data
+
+tests/save
+
+
+!tests/init/**/*
```

Comment:
Move these to the .gitignore in the root path.

Reply:
Fine